nengo_gui/static/netgraph.js

```diff
@@ -335,7 +335,7 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var uid = data.uid;
         for (var i = 0; i < Nengo.Component.components.length; i++) {
             if (Nengo.Component.components[i].uid === uid) {
-                Nengo.Component.components[i].remove(true);
+                Nengo.Component.components[i].remove(true, data.report_back);
```

Comment:
Lol, Terry you have the most visceral reactions to bad naming suggestions.

Now that you've explained it to me, as well as your methodology for naming parameters, it makes sense to me. If it's not too much trouble adding a quick comment might be a good idea, especially considering how deep it is in the if-statements.